src/routes/middleware/validators/validator.ts

```diff
@@ -0,0 +1,17 @@
+import { validationResult } from "express-validator";
+import { Request, Response, NextFunction } from "express";
+
+const validate = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
You can think about (you don't have to apply this if you don't want to; current solution is also ok) adding export before `const validate` and removing default export. Then in routes you would import this like `import { validate } from '...'`.

Some people prefer it this way because then IntelliSense provides you with hints what you can import and also you force the name "validate" in imports (unless explicitly changed).